exercises/practice/two-bucket/.docs/instructions.md

```diff
@@ -0,0 +1,46 @@
+# Instructions
+
+Given two buckets of different size and which bucket to fill first, determine how many actions are required to measure an exact number of liters by strategically transferring fluid between the buckets.
+
+There are some rules that your solution must follow:
+
+- You can only do one action at a time.
+- There are only 3 possible actions:
+  1. Pouring one bucket into the other bucket until either:
+     a) the first bucket is empty
+     b) the second bucket is full
+  2. Emptying a bucket and doing nothing to the other.
+  3. Filling a bucket and doing nothing to the other.
+- After an action, you may not arrive at a state where the starting bucket is empty and the other bucket is full.
```

Comment:
This instruction allows the starting bucket to be empty.  It says "may not" rather than "must not".  It should be that it does not allow (may I?) and must enforce that restriction.

There is a similar "may" later in the document as well, where it should be "must".